lib/matplotlib/axes/_base.py

```diff
@@ -3342,15 +3346,48 @@ def _set_view_from_bbox(self, bbox, direction='in',
         twiny : bool
             Whether this axis is twinned in the *y*-direction.
         """
+        Xmin, Xmax = self.get_xlim()
+        Ymin, Ymax = self.get_ylim()
+
+        if(len(bbox) == 3):
+            # Zooming code
+            xp, yp, scl = bbox
+            print("in set view, for zooming with scale {}".format(scl))
```

Comment:
I thought I had removed all of them. Removed it. As you can tell I didn't recompile and test after removing them. I confirm this time I recompiled and tested this last commit. Thanks!